packages/application-generic/src/services/in-memory-provider/providers/elasticache-cluster-provider.ts

```diff
@@ -52,7 +52,9 @@ export const getElasticacheClusterProviderConfig =
     };
 
     const host = redisClusterConfig.host;
-    const port = Number(redisClusterConfig.port);
+    const port = redisClusterConfig.port
```

Comment:
should we add a default port there or do we want it to fail if not provided?

Reply:
If no right configuration provided it will fallback to Redis Cluster configuration. If that configuration is not right nothing won't work and we report it in the logging.
Considering the Elasticache is our own implementation and the previous point I think we can not have a default. But open to suggestions.